### PR TITLE
tests: add GCE tests for diregapic

### PR DESCRIPTION
**Pull request by georgiyekkert**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-compute/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by georgiyekkert** [comment]
@alexander-fenster

File: system-test/.eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+env:
+  mocha: true
+rules:
+  node/no-unsupported-features/es-syntax: off
+  node/no-unsupported-features/node-builtins: off
+  no-console: off
```
Comment (alexander-fenster) [inline system-test/.eslintrc.yml:7]: nit: please add the new line :)

File: system-test/compute.js [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+  });
+
+  async function waitZonalOperation(operation) {
+    for (;;) {
+      const [getResp] = await operationsClient.get({
+        project,
+        zone,
+        operation: operation.name,
+      });
+      if (getResp.status === 2104194) {
```
Comment (alexander-fenster) [inline system-test/compute.js:304]: I'll need to figure out if there is a better way.  This is clearly unusable :)

**Comment by alexander-fenster** [comment]
OK so for the protos: can you use `compute.protos.google.cloud.compute.v1.Operation.Status.DONE` instead of the constant? To avoid typing that much:

```js
const Status = compute.protos.google.cloud.compute.v1.Operation.Status;
// then use Status.DONE
```

All values can be found in `protos/protos.d.ts`.

**Comment by alexander-fenster** [comment]
To make lint pass: `npx gts fix`  (can take a lot of time since the code is huge)

**Review by alexander-fenster — APPROVED** [review]

**Pull request merged**